M0nica/ambition-fund-website
# Fix: Add opportunities link to nav and make links more flexible

# Description

- Adds "opportunities" link to nav-list.
- Makes nav-links more flexible so they work on pages other than the home page.

## Related Issues



Fixes https://github.com/M0nica/ambition-fund-website/issues/11


## Pages/Interfaces that will change
Navigation

### Screenshots of Changes

![made-links-work](https://user-images.githubusercontent.com/42680433/85233960-2ccd5100-b3bf-11ea-8d00-ef41833d53f2.gif)

## Steps to test

1. Go to http://localhost:8000/ and click on the "opportunities" link. Should navigate to the opportunities page.
2. Click the "mission" link. This should navigate to the home page, scrolled to the "mission" section.
3. While on the home page, clicking the title or the "mission" links should smooth-scroll to the correct section.
